medusa/tv/series.py

```diff
@@ -1681,6 +1716,7 @@ def delete_show(self, full=False):
         # remove entire show folder
         if full:
             try:
+                _ = self.validate_location  # Let's get the exception out of the way asap.
```

Comment:
self.location does now by default all kind of checks on the filesystem. So if we use it in the apiv2, it will also do those checks there. In stead of having the option to just get the self.location (without any checks) and one optional that does the validations.

Reply:
ow wait, this line.. Pff i have to dig for that